src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private int randomInt = random.nextInt(100);

```

Comment:
If you declare your `randomInt` variable at class-level it will be the same value in `getRandomBall()` method on each call. You should declare this variable at method-level (local).
All magic numbers in your code should be constants. How do you think what max number will be returned by `random.nextInt(100);`?